atom/renderer/atom_renderer_client.cc

```diff
@@ -103,6 +103,12 @@ void AtomRendererClient::DidCreateScriptContext(
 
   // Setup node environment for each window.
   node::Environment* env = node_bindings_->CreateEnvironment(context);
+  auto* command_line = base::CommandLine::ForCurrentProcess();
+  // If we have disabled the site instance overrides we should prevent loading
+  // any non-context aware native module
+  if (command_line->HasSwitch(switches::kDisableElectronSiteInstanceOverrides))
+    env->ForceOnlyContextAwareNativeModules();
+
```

Comment:
`env->WarnNonContextAwareNativeModules()` is not needed for 6-0-x ?

Reply:
@deepak1556 Nope, the warning should only be present on `7-0-x` 👍